spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/model/Owner.java

```diff
@@ -45,83 +47,51 @@
  * @author Sam Brannen
  * @author Michael Isvy
  * @author Maciej Szarlinski
+ * @author Ramazan Sakin
  */
 @Entity
 @Table(name = "owners")
 public class Owner {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter
     private Integer id;
 
+    @Getter
+    @Setter
     @Column(name = "first_name")
-    @NotEmpty
+    @NotBlank
     private String firstName;
 
+    @Getter
+    @Setter
     @Column(name = "last_name")
-    @NotEmpty
+    @NotBlank
     private String lastName;
 
+    @Getter
+    @Setter
     @Column(name = "address")
-    @NotEmpty
+    @NotBlank
     private String address;
 
+    @Getter
+    @Setter
     @Column(name = "city")
-    @NotEmpty
+    @NotBlank
     private String city;
 
+    @Getter
+    @Setter
     @Column(name = "telephone")
-    @NotEmpty
-    @Digits(fraction = 0, integer = 10)
+    @NotBlank
+    @Digits(fraction = 0, integer = 12)
```

Comment:
I notice you increase telephone length from 10 to 12. It's OK for me because in the `schelma.sql` we used a VARCHAR(20):
```
 telephone  VARCHAR(20)
```